From 2000f4c135c236457be98cc837abcb59327fd22a Mon Sep 17 00:00:00 2001
From: Jeremie GARCIA <jeremiex.garcia@intel.com>
Date: Fri, 24 May 2013 15:47:36 +0200
Subject: [PORT FROM MAIN]wifi: temp fix for netd build issues

BZ: 117199

Temp fix for netd build issues

Change-Id: Ie95634c139fb0b9033c6dbf37840db6f4198fc6f
Orig-Change-Id: I6501dbb1984b0245281691ebb062a00a3ab5abc2
Category: device enablement
Domain: CWS.WIFI-Intel
Origin: internal
Upstream-Candidate: yes
Signed-off-by: Jeremie GARCIA <jeremiex.garcia@intel.com>
Signed-off-by: Jeremie GARCIA <jeremie.garcia@intel.com>
---
 Android.mk |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Android.mk b/Android.mk
index ed0238a..23ba575 100644
--- a/Android.mk
+++ b/Android.mk
@@ -64,6 +64,7 @@ LOCAL_MODULE_STEM := netd
 LOCAL_CFLAGS :=
 LOCAL_SRC_FILES:=                                      \
                   BandwidthController.cpp              \
+                  ClatdController.cpp                  \
                   CommandListener.cpp                  \
                   DnsProxyListener.cpp                 \
                   FirewallController.cpp               \
@@ -79,9 +80,8 @@ LOCAL_SRC_FILES:=                                      \
                   ResolverController.cpp               \
                   SecondaryTableController.cpp         \
                   TetherController.cpp                 \
-                  ThrottleController.cpp               \
                   oem_iptables_hook.cpp                \
-                  logwrapper.c                         \
+                  UidMarkMap.cpp                       \
                   main.cpp                             \
 
 LOCAL_SRC_FILES += SoftapController.cpp
-- 
1.7.4.1

